src/cartesian/Funnel.tsx

```diff
@@ -192,9 +189,9 @@ const getRealWidthHeight = ({ customWidth }: { customWidth: number | string }, o
   const realHeight = height;
   let realWidth = width;
 
-  if (isNumber(customWidth)) {
+  if (typeof customWidth === 'number') {
```

Comment:
I don't think this matters that much but these functions are not 1:1 https://stackoverflow.com/a/56664032/8050147

But since this is in a new major version this is probably fine

Reply:
Can do the following instead, that'll check for the object type as well.

```js
typeof customWidth === 'number' || customWidth instanceof Number
```

Reply:
That gets lengthy when repeated a lot if you'd like to make a small utility for that too

Reply:
Would this utility not be fine? It also makes sure the value is a valid number 
https://github.com/KSJaay/recharts/blob/9856e7ab8f82771a8e157b0c09e11ee1cfa02e83/src/util/DataUtils.ts#L17

Reply:
Oh yeah that works. Hard to keep track of them sometimes 😅